git/repo/base.py

```diff
@@ -429,11 +468,16 @@ def _get_config_path(self, config_level):
         elif config_level == "global":
             return osp.normpath(osp.expanduser("~/.gitconfig"))
         elif config_level == "repository":
-            return osp.normpath(osp.join(self._common_dir or self.git_dir, "config"))
```

Comment:
It's probably nicer to do it like this:

```python
if not (self._common_dir or self.git_dir):
    raise …
return osp.normpath(osp.join(self._common_dir or self.git_dir, "config"))
```

That way there is no duplicate path transformation.

Reply:
amended in #1202